server/belga/image.py

```diff
@@ -212,6 +212,93 @@ def format_list_item(self, data):
         }
 
 
+class Belga360ArchiveSearchProvider(superdesk.SearchProvider):
```

Comment:
I think `image.py` file is a wrong place for a text archive search provider.
IMO it makes sense to rename `image.py` to `search_providers.py` in this case.